pages/common/nms.md

```diff
@@ -0,0 +1,20 @@
+# nms
+
+> A cool (piped) text decryption effect.
+> More information: <https://github.com/bartobri/no-more-secrets>.
+
+- Decrypt "Hello, World!" after a keystroke:
+
+`echo "Hello, World! | nms"`
+
+- Decrypt the output of `ls -la` automatically:
+
+`ls -la | nms -a`
+
+- Decrypt the content of `secret_message.txt` automatically with red output:
+
+`cat secret_message.txt | nms -a -f red`
```

Comment:
If the list is short enough, include the whole list of valid values separated by `|` (I put the default `blue` first, otherwise thie order is that found in the man page):
```suggestion
`cat {{path/to/file}} | nms -a -f {{blue|white|yellow|black|magenta|green|red}}`
```